From faf4d1308f89560b0d849671226e3080dfc72e79 Mon Sep 17 00:00:00 2001
From: Anthony Fok <foka@debian.org>
Date: Sat, 5 Sep 2015 04:42:59 -0600
Subject: [PATCH] %Import QtXml/QtXmlmod.sip if available

to fix `sip: QDomElement is undefined` error.

Fixes #2
---
 poppler-qt5.sip | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/poppler-qt5.sip b/poppler-qt5.sip
index e89cde3..6744eae 100644
--- a/poppler-qt5.sip
+++ b/poppler-qt5.sip
@@ -5,11 +5,13 @@
  * Maintained by Wilbert Berendsen <wbsoft@xs4all.nl>
  */
 
+%Feature QTXML_AVAILABLE
 
 %Import QtCore/QtCoremod.sip
 %Import QtGui/QtGuimod.sip
-
-%Feature QTXML_AVAILABLE
+%If(QTXML_AVAILABLE)
+%Import QtXml/QtXmlmod.sip
+%End
 
 %Timeline {
     POPPLER_V0_20_0
